src/views/order_view.js

```diff
@@ -0,0 +1,86 @@
+import Backbone from 'backbone';
+import Order from '../models/order';
+
+const OrderView = Backbone.View.extend({
+  initialize(params) {
+    this.template = params.template;
+    this.bus = params.bus;
+
+    this.listenTo(this.model, 'change', this.render);
+    this.listenTo(this.bus, 'price_change', this.checkForPriceMatch);
+  },
+
+  checkForPriceMatch(model) {
+    console.log('In checkForPriceMatch');
+    console.log(model);
+
+    if (model.get('symbol') === this.model.get('symbol')) {
+      console.log('Symbol Match');
+
+      if (this.model.get('buy')) {
+        console.log('Buy attribute is true');
+
+        if (this.model.get('targetPrice') >= model.get('price').toFixed(2)) {
+          console.log('Executing buy order');
+
+          const objectForTradeHistory = {
+            model: model,
+            buy: true,
+            price: parseFloat(this.model.get('targetPrice')),
+            symbol: this.model.get('symbol'),
+          };
+
+          this.bus.trigger('add_me_to_trade_hist', objectForTradeHistory);
+          this.stopListening(); //destroy alone doesn't stop the listener
+          this.model.destroy();
+        }
+      } else {
+        console.log('Buy attribute is false');
+
+        if (this.model.get('targetPrice') <= model.get('price').toFixed(2)) {
```

Comment:
This conditional is not going to work how you expect, because we're comparing two strings instead of two numbers:
* `this.model.get('targetPrice')` will be a string because when we build the form data into an `Order` model instance we do not parse the price input field's value with `Number.parseFloat`.
* `model.get('price').toFixed(2)` will be a string because when you call `toFixed()` it returns a string.

The result is that we're doing a _lexical comparison_ -- that is, we're comparing each digit as a separate character. This means that `"10000" < "20"` is true, because "1" is considered less than "2".